src/main/resources/assets/ematter/models/block/seebeck_generator.json

```diff
@@ -0,0 +1,12 @@
+{
+  "parent": "block/cube",
+  "textures": {
+    "particle": "ematter:blocks/seebeck_generator/side",
+    "up": "ematter:blocks/seebeck_generator/up",
+    "down": "ematter:blocks/seebeck_generator/up",
+    "north": "ematter:blocks/seebeck_generator/side",
+    "east": "ematter:blocks/seebeck_generator/side",
+    "south": "ematter:blocks/seebeck_generator/side",
+    "west": "ematter:blocks/seebeck_generator/side"
+  }
+}
```

Comment:
Missing newline at end of file (though yet again this is temporary, it'll be replaced with a BlockBench alternative so...)